pinot-core/src/main/java/org/apache/pinot/core/common/MinionConstants.java

```diff
@@ -62,16 +62,35 @@ private MinionConstants() {
     public static final String MERGED_SEGMENT_NAME_KEY = "mergedSegmentNameKey";
   }
 
+  /**
+   * Creates segments for the OFFLINE table, using completed segments from the corresponding REALTIME table
+   */
   public static class RealtimeToOfflineSegmentsTask {
     public static final String TASK_TYPE = "realtimeToOfflineSegmentsTask";
-    // window
+
+    /**
+     * The time window size for the task.
+     * e.g. if set to "1d", then task is scheduled to run for a 1 day window
+     */
+    public static final String BUCKET_TIME_PERIOD_KEY = "bucketTimePeriod";
+    /**
+     * The time period to wait before picking segments for this task
+     * e.g. if set to "2d", no task will be scheduled for a time window younger than 2 days
```

Comment:
```suggestion
     * e.g. if set to "2d", no task will be scheduled for a segments newer than 2 days
```
:)
Is this interpretation correct?